qiita_client/tests/test_qiita_client.py

```diff
@@ -158,80 +132,69 @@ def test_patch_value_error(self):
         # Add, replace or test
         with self.assertRaises(ValueError):
             self.tester.patch(
-                '/qiita_db/artifacts/1/filepaths/', 'add', '/html_summary/',
+                '/qiita_db/artifacts/1/', 'add', '/html_summary/',
```

Comment:
The filepaths endpoint no longer exists. In any case, this is just checking the `patch` functionality of the qiita_client, so the only thing that I needed is an endpoint that allows patching.